hudi-common/src/main/java/org/apache/hudi/common/model/HoodieFileGroup.java

```diff
@@ -103,12 +106,30 @@ public void addBaseFile(HoodieBaseFile dataFile) {
    * Add a new log file into the group.
    */
   public void addLogFile(HoodieLogFile logFile) {
-    if (!fileSlices.containsKey(logFile.getBaseCommitTime())) {
-      fileSlices.put(logFile.getBaseCommitTime(), new FileSlice(fileGroupId, logFile.getBaseCommitTime()));
+    String baseInstantTime = getBaseInstantTime(logFile);
```

Comment:
changes to accomodate for file slice determination for both 0.x and 1x log files.